torch/ao/quantization/fake_quantize.py

```diff
@@ -347,7 +348,7 @@ def forward(self, X: torch.Tensor) -> torch.Tensor:
 """
 
 default_dynamic_fake_quant = FakeQuantize.with_args(observer=MovingAverageMinMaxObserver, quant_min=0, quant_max=255,
-                                                    dtype=torch.quint8, averaging_constant=1)
+                                                    dtype=torch.quint8, averaging_constant=1, is_dynamic=True)
```

Comment:
if we inherit this from observer I think we can remove this flag from fake quantize api

Reply:
should be configured like this:

FakeQuantize.with_args(observer=MovingAverageMinMaxObserver.with_args(is_dynamic=True), ...)

Reply:
I think all arguments aside from quant_min and quant_max are passed to the observer via observer kwargs here: https://www.internalfb.com/code/fbsource/[205ceaf0d5c23e4fdc62814a3622c745703e6b58]/fbcode/caffe2/torch/ao/quantization/fake_quantize.py?lines=140

since things like averaging_constant are passed via that method I figured this would be fine. It seems weird to me to pass 1 via that method and another by directly modifying the observer argument.

Reply:
I see, maybe we can postpone this, but I feel the final state should be we just modify the config for observer directly